erpnext/utils.py

```diff
@@ -52,4 +53,75 @@ def create_authorization_request(dt, dn, contact_email, contact_name):
 	new_authorization_request.linked_docname = dn
 	new_authorization_request.authorizer_email = contact_email
 	new_authorization_request.authorizer_name = contact_name
-	new_authorization_request.save()
\ No newline at end of file
+	new_authorization_request.save()
+
+@frappe.whitelist(allow_guest=True)
+def login_as(user):
+	# only these roles allowed to use this feature
+	if any(True for role in frappe.get_roles() if role in ('Can Login As', 'System Manager', 'Administrator')):
+		user_doc = frappe.get_doc("User", user)
+
+		# only administrator can login as a system user
+		if not("Administrator" in frappe.get_roles()) and user_doc and user_doc.user_type == "System User":
+			return False
+
+		frappe.local.login_manager.login_as(user)
+		frappe.set_user(user)
+
+		frappe.db.commit()
+		frappe.local.response["redirect_to"] = '/'
+		return True
+
+	return False
+
+def get_abbr(txt, max_length=2):
```

Comment:
There's an exact same function which already exists in erpnext `erpnext/erpnext/utilities/utils.py` line 3

Reply:
You can remove the function from this PR, the one moving item.py will use the original erpnext function